sunpy/instr/rhessi.py

```diff
@@ -27,6 +34,51 @@
 lc_linecolors = ('black', 'pink', 'green', 'blue', 'brown', 'red',
                  'navy', 'orange', 'green')
 
+known_flare_list_sources = {
+    "NASA": "https://hesperia.gsfc.nasa.gov/hessidata/dbase/",
+    "i4DS": "http://soleil.i4ds.ch/hessidata/dbase/",
+}
+
+flagid2flag = {
```

Comment:
hm. I named them after the existing `grid_pitch`, `grid_orientation` and `lc_linecolors` ... I suggest to either change all or none to uppercase.. or are they somehow different?

on the other hand, changing them could break behaviour for users.. so if you'd prefer to leave them as-is but still introduce the new ones as all-caps I guess that would be somewhat justified.

Reply:
I think it would be good to at least change the ones you created. 

A quick search of the code show these are only used from this class and I can't see these also they are not in `__all__` so not exposed also I can see these being  used directly so I would change the rest too, but maybe wait for a second opinion on that.

Reply:
Just a quick note, as long as its defined in the code, they are accessible (aka you can import them directly if you look for it). The all just hides them if you do import *.